### ISSUE TITLE
Error Message: REST API is not yet supported for this mailbox.

### ISSUE THREAD

**Issue by salihzett** [O365/python-o365#438]
`Client Error: Not Found for url: https://graph.microsoft.com/v1.0/me/calendar/events | Error Message: REST API is not yet supported for this mailbox.`
For emailing or calendar. Is there any way to change this to v2.0?

**Comment by janscas** [comment]
yes, look for protocols in the readme. there you can change the api version

**Comment by Micraow** [comment]
也有可能是因为你没有给那个邮箱许可证。
